internals/core_models.py

```diff
@@ -954,3 +954,206 @@ def put(self, notify=True, **kwargs):
   anticipated_spec_changes = ndb.StringProperty()
 
   finch_url = ndb.StringProperty()
+
+
+# Note: This class is not used yet.
+class FeatureEntry(ndb.Model):  # Copy from Feature
+  """This is the main representation of a feature that we are tracking."""
+
+  # Metadata: Creation and updates.
+  created = ndb.DateTimeProperty(auto_now_add=True)
+  updated = ndb.DateTimeProperty()
+  accurate_as_of = ndb.DateTimeProperty()
+  creator = ndb.StringProperty()
+  updater = ndb.StringProperty()
+
+  # Metadata: Access controls
+  owners = ndb.StringProperty(repeated=True)  # copy from owner
+  editors = ndb.StringProperty(repeated=True)
+  unlisted = ndb.BooleanProperty(default=False)
+  deleted = ndb.BooleanProperty(default=False)
+
+  # Descriptive info.
+  name = ndb.StringProperty(required=True)
+  summary = ndb.TextProperty(required=True)
+  category = ndb.IntegerProperty(required=True)
+  blink_components = ndb.StringProperty(repeated=True)
+  star_count = ndb.IntegerProperty(default=0)
+  search_tags = ndb.StringProperty(repeated=True)
+  feature_notes = ndb.TextProperty()  # copy from comments
+
+  # Metadata: Process information
+  feature_type = ndb.IntegerProperty(default=FEATURE_TYPE_INCUBATE_ID)
+  intent_stage = ndb.IntegerProperty(default=0)
+  bug_url = ndb.StringProperty()
+  launch_bug_url = ndb.StringProperty()
+
+  # Implementation in Chrome
+  impl_status_chrome = ndb.IntegerProperty(required=True)
+  flag_name = ndb.StringProperty()
+  ongoing_constraints = ndb.TextProperty()
+
+  # Gate: Adoption
+  motivation = ndb.TextProperty()
+  devtrial_instructions = ndb.TextProperty()
+  activation_risks = ndb.TextProperty()
+  measurement = ndb.TextProperty()
+
+  # Gate: Standardization & Interop
+  initial_public_proposal_url = ndb.StringProperty()
+  explainer_links = ndb.StringProperty(repeated=True)
+  requires_embedder_support = ndb.BooleanProperty(default=False)
+  standard_maturity = ndb.IntegerProperty(required=True, default=UNSET_STD)
+  spec_link = ndb.StringProperty()
+  api_spec = ndb.BooleanProperty(default=False)
+  spec_mentors = ndb.StringProperty(repeated=True)
+  interop_compat_risks = ndb.TextProperty()
+  prefixed = ndb.BooleanProperty()
+  all_platforms = ndb.BooleanProperty()
+  all_platforms_descr = ndb.TextProperty()
+  tag_review = ndb.StringProperty()
+  tag_review_status = ndb.IntegerProperty(default=REVIEW_PENDING)
+  non_oss_deps = ndb.TextProperty()
```

Comment:
Here's [the only feature](https://chromestatus.com/feature/5759984304390144) where this has a value other than blank or 'none'. I can't believe there aren't more than that. 

We should start a document that lists who uses every field in the system to keep track of what's still needed and what isn't.